crates/cli-tools/src/changelog.rs

```diff
@@ -173,6 +173,21 @@ impl Changelog {
 
         Ok(changelog)
     }
+
+    fn validate_cargo_toml(&self, path: &str) -> Result<()> {
+        let most_recent_release =
+            self.releases.first().context("At least 1 release is required")?;
```

Comment:
Can this happen? We only produce valid changelogs right? So it's an invariant that `releases` is not empty.

Reply:
Correct, that cannot happen. I'll use `unwrap()` instead.